[2024-12-09T20:46:10.521+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-09T20:46:10.538+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_data_pipeline_mongodb_complex.branch_task manual__2024-12-09T20:45:56.898218+00:00 [queued]>
[2024-12-09T20:46:10.548+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_data_pipeline_mongodb_complex.branch_task manual__2024-12-09T20:45:56.898218+00:00 [queued]>
[2024-12-09T20:46:10.548+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-09T20:46:10.561+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): branch_task> on 2024-12-09 20:45:56.898218+00:00
[2024-12-09T20:46:10.571+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=871) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-09T20:46:10.573+0000] {standard_task_runner.py:72} INFO - Started process 873 to run task
[2024-12-09T20:46:10.573+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_data_pipeline_mongodb_complex', 'branch_task', 'manual__2024-12-09T20:45:56.898218+00:00', '--job-id', '322', '--raw', '--subdir', 'DAGS_FOLDER/etl_data_pipeline_mongodb_complex.py', '--cfg-path', '/tmp/tmpd_0vaoej']
[2024-12-09T20:46:10.575+0000] {standard_task_runner.py:105} INFO - Job 322: Subtask branch_task
[2024-12-09T20:46:10.622+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_data_pipeline_mongodb_complex.branch_task manual__2024-12-09T20:45:56.898218+00:00 [running]> on host 8553af9ff67b
[2024-12-09T20:46:10.931+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_data_pipeline_mongodb_complex' AIRFLOW_CTX_TASK_ID='branch_task' AIRFLOW_CTX_EXECUTION_DATE='2024-12-09T20:45:56.898218+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-09T20:45:56.898218+00:00'
[2024-12-09T20:46:10.933+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-09T20:46:10.933+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-09T20:46:10.934+0000] {logging_mixin.py:190} INFO - Current task name:branch_task state:running start_date:2024-12-09 20:46:10.539497+00:00
[2024-12-09T20:46:10.934+0000] {logging_mixin.py:190} INFO - Dag name:etl_data_pipeline_mongodb_complex and current dag run status:running
[2024-12-09T20:46:10.935+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-09T20:46:10.936+0000] {baseoperator.py:416} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-12-09T20:46:10.950+0000] {python.py:240} INFO - Done. Returned value was: load_data
[2024-12-09T20:46:10.950+0000] {branch.py:38} INFO - Branch into load_data
[2024-12-09T20:46:10.951+0000] {skipmixin.py:230} INFO - Following branch ('load_data',)
[2024-12-09T20:46:10.963+0000] {skipmixin.py:278} INFO - Skipping tasks [('skip_load', -1)]
[2024-12-09T20:46:10.997+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-09T20:46:10.998+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_data_pipeline_mongodb_complex, task_id=branch_task, run_id=manual__2024-12-09T20:45:56.898218+00:00, execution_date=20241209T204556, start_date=20241209T204610, end_date=20241209T204610
[2024-12-09T20:46:11.007+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-12-09T20:46:11.008+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-12-09T20:46:11.008+0000] {logging_mixin.py:190} INFO - Dag name:etl_data_pipeline_mongodb_complex queued_at:2024-12-09 20:45:56.905278+00:00
[2024-12-09T20:46:11.008+0000] {logging_mixin.py:190} INFO - Task hostname:8553af9ff67b operator:BranchPythonOperator
[2024-12-09T20:46:11.029+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-09T20:46:11.237+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-09T20:46:11.239+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
